src/modules/scores.js

```diff
@@ -1,6 +1,7 @@
 export default class Scores {
-  constructor(name, score) {
-    this.name = name;
-    this.score = score;
+  constructor(userName, points) {
+    userName = this.userName;
+    // eslint-disable-next-line no-unused-vars
+    points = this.points;
   }
```

Comment:
While you did a wonderful job workimg on the requirements I noticed you disabled eslint. It is against microverse policy to disable linters because linters help us write clean code. It would be nice if you enabled eslint